submissions/rafiatAminu/html-forms/contact.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact Form</title>
+</head>
+
+<body>
+    <h1>RAF PHONE STORES</h1>
+    <nav>
+        <ul>
+            <li><a href="#">Store</a></li>
+            <li><a href="#">Contact</a></li>
+            <li><a href="#">Cart</a></li>
+        </ul>
+    </nav>
+    <hr>
+    <h1>Contact Form</h1>
+    <form action="#" method="POST" enctype="multipart/form-data">
+        <fieldset>
+            <legend>Contact Form</legend>
+            <label for="firstname">First Name</label><br>
+            <input type="text" name="firstname" id="firstname" /><br>
+            <label for="lastname">Last Name</label><br>
+            <input type="text" name="lastname" id="lastname" /><br>
+            <label for="email">Enter your email address</label><br>
+            <input type="text" name="email" id="email" /><br>
+            <label for="subject">Subject</label><br>
+            <input type="text" name="firstname" id="firstname" /><br>
+            <label for="message">Message Input</label><br>
+            <textarea name="message" id="message" cols="50" rows="10"></textarea><br>
+
```

Comment:
Remove whitespace

Reply:
![Screen Shot 2020-12-08 at 9 14 26 AM](https://user-images.githubusercontent.com/46797474/101457809-47c45b00-3936-11eb-953f-9d015e2d0482.png)

Here's a screenshot of my code, there's no whitespace. I have formatted countless times